### DISCUSSION TITLE
1.79 release - when it can be expected?

### DISCUSSION THREAD

**Discussion by dudaerich** [bcgit/bc-java#1849] [General]
Hi everyone,
we are trying to upgrade to the newest version of Bouncy castle but we are blocked by issue #1641. We prepared a workaround in PR #1771 which was also merged, but we would like to know when we can expect that the workaround will be released.

Is there any plan for 1.79 release? Thank you.

**Comment by dghgit** [comment]
End of October.

**Comment by dghgit** [comment]
It came out on the 30th!
